src/coreclr/jit/codegencommon.cpp

```diff
@@ -2588,7 +2586,6 @@ void CodeGen::genReportEH()
                 }
                 else
                 {
-                    assert(bbLabel->bbEmitCookie != nullptr);
```

Comment:
why is this removed?

Reply:
The call to `ehCodeOffset` has the same assert, so it didn't seem necessary.